tools/sgen/input/FluentUIStyle.yml

```diff
@@ -190,28 +310,66 @@ AP_MSFAvatarTokens:
   ringThickness: [ xSmall: $Border.size.thin, small: $Border.size.thin, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
   ringOuterGap: [ xSmall: $Border.size.thick, small: $Border.size.thick, medium: $Border.size.thick, large: $Border.size.thick, xlarge: $Border.size.thick, xxlarge: $Border.size.thicker ]
   size: [ xSmall: 16pt, small: 24pt, medium: 32pt, large: 40pt, xlarge: 52pt, xxlarge: 72pt ]
-  textCalculatedBackgroundColors: [ "NamedColor(cyanBlue10)",
-                                    "NamedColor(red10)",
-                                    "NamedColor(magenta20)",
-                                    "NamedColor(green10)",
-                                    "NamedColor(magentaPink10)",
-                                    "NamedColor(cyanBlue20)",
-                                    "NamedColor(orange20)",
-                                    "NamedColor(cyan20)",
-                                    "NamedColor(orangeYellow20)",
-                                    "NamedColor(red20)",
-                                    "NamedColor(blue10)",
-                                    "NamedColor(magenta10)",
-                                    "NamedColor(gray40)",
-                                    "NamedColor(green20)",
-                                    "NamedColor(blueMagenta20)",
-                                    "NamedColor(pinkRed10)",
-                                    "NamedColor(gray30)",
-                                    "NamedColor(blueMagenta30)",
-                                    "NamedColor(gray20)",
-                                    "NamedColor(cyan30)",
-                                    "NamedColor(orange30)" ]
-  textColor: $Colors.Foreground.neutralInverted
+  textCalculatedBackgroundColors: [ "FluentUIColor(light: $Colors.DarkRed.tint40, dark: $Colors.DarkRed.shade30)",
```

Comment:
Should we do `textCalculatedColor: [ background: <> , foreground: <>]` instead to enforce mapping ?

Reply:
We'd need 30 items of "[ background: <> , foreground: <>]" (which falls into the third level of hierarchy that SGen does not support).